Sources/Resources/Sentry.modulemap

```diff
@@ -3,4 +3,32 @@ framework module Sentry {
 
     export *
     module * { export * }
+    
+    explicit module _Hybrid {
+        // Headers that are not part of the public API and should only be used by hybrid SDKs
+        header "PrivateSentrySDKOnly.h"
+        header "PrivatesHeader.h"
+        header "SentryAppStartMeasurement.h"
+        header "SentryBinaryImageCache.h"
+        header "SentryBreadcrumb+Private.h"
+        header "SentryDebugImageProvider+HybridSDKs.h"
+        header "SentryDependencyContainer.h"
+        header "SentryEnvelope.h"
+        header "SentryEnvelopeItemType.h"
+        header "SentryFormatter.h"
+        header "SentryFramesTracker.h"
+        header "SentryOptions+HybridSDKs.h"
+        header "SentryScreenFrames.h"
+        header "SentrySwizzle.h"
+        header "SentryUser+Private.h"
+
+        header "SentryBaseIntegration.h"
+        header "SentrySessionReplayIntegration.h"
+        header "SentrySessionReplayIntegration-Hybrid.h"
+        
+        header "SentrySdkInfo.h"
+        header "SentryInternalSerializable.h"
```

Comment:
l: Is it possible to have one header that has all this other headers as reference? The same way we do with `Sentry.h`.

Just because a header can be more versatile.

Reply:
Had some issues with this approach, but will try again.

Reply:
I always get an error about non-modular headers when i try to move the header imports into one header.

<img width="1687" alt="Bildschirmfoto 2024-11-13 um 14 10 00" src="https://github.com/user-attachments/assets/aaef3daf-94ff-464b-b1d2-44eecd12b546">

Reply:
Ok, no problem. We can continue without it.